locales/cy/translations.json

```diff
@@ -128,6 +128,19 @@
         "errorStartAddressMissing" : "Rhowch gyfeiriad"
     },
 
+    "whatIsYourJurisdiction" : {
+        "title" : "WELSH - What is your jurisdiction?",
+        "whereIsJurisdiction" : "WELSH - Where is the jurisdiction?",
+        "options": {
+            "EnglandWales" : "WELSH - England and Wales",
+            "NorthernIreland" : "WELSH - Northern Ireland"
+        },
+        "hint" : "WELSH - This is where the partnership was formed. The registered office address and principal place of business will need to be in the same jurisdiction.", 
+        "scotlandHeading" : "WELSH - This partnership will be registered in Scotland",
+        "scotlandInformation" : "WELSH - This means the partnership's registered office address and principal place of business will need to be in Scotland. Select to continue, if this is correct .",
```

Comment:
just noticed there is an extra space before the full stop in scotlandInformation (both en and cy files)
"Select to continue, if this is correct ."
don't think that space should be before the final full stop.

Reply:
Removed extra space.